buildSrc/src/main/kotlin/io/realm/RealmPublishPlugin.kt

```diff
@@ -82,6 +94,7 @@ class RealmPublishPlugin : Plugin<Project> {
             configureRootProject(project)
         } else {
             configureSubProject(project, signBuild)
+            configureCustomRepository(project)
```

Comment:
Can we guard this by the CI environment since the repo is already consumed when building from CI

Reply:
All this does is setting up new publication targets, so it doesn't really impact either CI or local development unless you call them (which right now only CI does). But I think we should keep it for both cases, just in case of debugging.